kernels/streamer_matmul/main.c

```diff
@@ -28,10 +28,14 @@
 #define meshCol 8
 
 uint8_t Batch = 1;
-// meshRow, tileSize and meshCol are defined in snax-gemm-params.h
-uint8_t M_param = M_size / meshRow;
+
+/* M_param and N_param can be set to 1 for tiled versions, but not for simple
+ version. 2 always works. however, it will impact performance significantly as
+ computation cost doubles. For benchmarks, set to 1 */
+uint8_t M_param = 2;
 uint8_t K_param = K_size / tileSize;
-uint8_t N_param = N_size / meshCol;
+uint8_t N_param = 2;
+
```

Comment:
This is a bit fishy but should get resolved when we compile for this kernel from within MLIR!